docs/docsite/rst/user_guide/windows_dsc.rst

```diff
@@ -222,10 +236,10 @@ The ``Find-DscResource`` cmdlet can also be used to find custom resources. For e
 
 .. code-block:: powershell
 
-    # find all DSC resources in the configured repositories
+    # Find all DSC resources in the configured repositories
```

Comment:
Same thing here, is it really necessary we do this?

Reply:
Is it necessary ? From a technical perspective no, but to me it's more pleasing.

But I am the guy who prefers punctuation marks and correct spelling too ;-)